lib/iris/fileformats/netcdf.py

```diff
@@ -2104,9 +2194,19 @@ def save(cube, filename, netcdf_format='NETCDF4', local_keys=None,
     with Saver(filename, netcdf_format) as sman:
         # Iterate through the cubelist.
         for cube in cubes:
+            if isinstance(packing, list):
```

Comment:
You can test for an iterable with

```python
from collections import Iterable

if isinstance(some_thing, Iterable):
    ...
```

This will return `True` for string also, so you'll need to check if it is iterable, but not a string.